File: python/Easy/ExcelSheetColumnTitle.py
class Solution:
    def convertToTitle(self, c: int) -> str:
        r = ""
        while c > 0:
            c -= 1
            r = chr((c%26) + 65) + r
            c //= 26
        return r
    
s = Solution()
print(s.convertToTitle(28))

File: javascript/Easy/ExcelSheetColumnTitle.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function(columnNumber) {
    let r = "";
    while (columnNumber > 0) {
        columnNumber--;
        r = String.fromCharCode(columnNumber%26 + 65) + r;
        columnNumber = parseInt(columnNumber/26);
    }
    return r;
};

console.log(convertToTitle(53))